central/activecomponent/datastore/internal/store/postgres/store.go

```diff
@@ -110,11 +110,11 @@ func insertIntoActiveComponents(ctx context.Context, batch *pgx.Batch, obj *stor
 	return nil
 }
 
-func insertIntoActiveComponentsActiveContextsSlices(_ context.Context, batch *pgx.Batch, obj *storage.ActiveComponent_ActiveContext, active_components_Id string, idx int) error {
+func insertIntoActiveComponentsActiveContextsSlices(_ context.Context, batch *pgx.Batch, obj *storage.ActiveComponent_ActiveContext, activeComponentsID string, idx int) error {
```

Comment:
`activeComponentsID` -> `activeComponentID` and same comment for all singular ID fields.